src/main/java/org/ow2/proactive/procci/model/occi/infrastructure/Network.java

```diff
@@ -85,8 +85,8 @@ private Network(Optional<String> url, Kind kind, Optional<String> title, List<Mi
         this.label = label;
     }
 
-    public static Set<Attribute> getAttributes() {
-        Set<Attribute> attributes = Resource.getAttributes();
+    public static Set<Attribute> createAttributeSet() {
```

Comment:
there is a design problem here with this method, which is static and exists in several classes. not clear why it must be static, and why in some classes, it is public, some classes, it is private. Could it be modified as a protected method in the parent abstract class, and the extended classes can simply override it ? We should not have createAttributeSet in an abstract Entity class, since it is abstract class, it does not make sense